eth_account/hdaccount/mnemonic.py

```diff
@@ -0,0 +1,178 @@
+# Originally from: https://github.com/trezor/python-mnemonic
+#
+# Copyright (c) 2013 Pavol Rusnak
+# Copyright (c) 2017 mruddy
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+# of the Software, and to permit persons to whom the Software is furnished to do
+# so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+import binascii
+import hashlib
+import os
+from pathlib import (
+    Path,
+)
+import unicodedata
+
+from eth_utils import (
+    ValidationError,
+    combomethod,
+)
+
+PBKDF2_ROUNDS = 2048
+VALID_SEED_SIZES = [16, 20, 24, 28, 32]
+VALID_WORD_LENGTHS = [12, 15, 18, 21, 24]
```

Comment:
nit: `LENGTHS` makes me think of string/letter count of each word; thoughts on `COUNTS` instead? The spec calls the collection of words a `mnemonic sentence`, so another alternative: `VALID_SENTENCE_LENGTHS`. non-blocking.

Reply:
`COUNTS` is fine by me

Reply:
FIxed with 8e74421